src/components/modules/BucketSettings/index.js

```diff
@@ -69,7 +123,7 @@ const BucketSettings = ({
                   open={showConfirm}
                   cancelButton="Cancel"
                   confirmButton="Delete"
-                  onCancel={closeConfirmDelete}
+                  onCancel={() => setShowConfirm(false)}
```

Comment:
Any reason this was changed from a method to an arrow function?

Reply:
I was thinking that since that the only spot where that function was used it could be an arrow function. Also since it was calling one function.